<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="ru_RU">
<defaultcodec>UTF-8</defaultcodec>
<context>
    <name>QObject</name>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="139"/>
        <source>OK</source>
        <translation>ОК</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="148"/>
        <source>Bad Request</source>
        <translation>Плохой запрос</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="151"/>
        <source>Unauthorized</source>
        <translation>Неавторизован</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="154"/>
        <source>Forbidden</source>
        <translation>Запрещено</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="157"/>
        <source>Not Found</source>
        <translation>Не найдено</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="166"/>
        <source>Nick Already In Use</source>
        <translation>Ник уже используется</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="160"/>
        <source>Object Already Exists</source>
        <translation>Объект уже существует</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="142"/>
        <source>Found</source>
        <translation>Найдено</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="145"/>
        <source>Not Modified</source>
        <translation>Не изменялось</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="163"/>
        <source>Object Not Exists</source>
        <translation>Объект  не существует</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="169"/>
        <source>Channel Offline</source>
        <translation>Канал не в сети</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="172"/>
        <source>Conflict</source>
        <translation>Конфликт</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="175"/>
        <source>Internal Error</source>
        <translation>Внутренняя ошибка</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="178"/>
        <source>Not Implemented</source>
        <translation>Не реализовано</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="181"/>
        <source>Bad Gateway</source>
        <translation>Плохой шлюз</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="184"/>
        <source>Service Unavailable</source>
        <translation>Сервис недоступен</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="187"/>
        <source>Gateway Timeout</source>
        <translation>Шлюз не отвечает</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/Notice.cpp" line="190"/>
        <source>Unknown</source>
        <translation>Неизвестная ошибка</translation>
    </message>
</context>
<context>
    <name>SimpleSocket</name>
    <message>
        <location filename="../../src/common/net/SimpleSocket.cpp" line="205"/>
        <source>Time out</source>
        <translation>Время вышло</translation>
    </message>
</context>
</TS>
